svc-lighthouse-api/src/test/java/gov/va/vro/abddataaccess/service/FhirClientTest.java

```diff
@@ -123,116 +134,101 @@ private void mockRest(ResponseEntity<String> resp, String domainName) {
   }
 
   @Test
-  public void testGetMedicalEvidence() {
+  public void testGetMedicalEvidence() throws AbdException {
```

Comment:
I haven't tested the following case. If the exception is thrown, will the failed test be counted in the test result html page?

Reply:
Yes. You can just try a test which only has a single line "throw new AbdEvidence("error");" and it will fail. All the assertions throw exceptions. That is how they fail.